src/backend/legendary/games.ts

```diff
@@ -502,6 +505,24 @@ class LegendaryGame extends Game {
       return { status: 'error', error: res.error }
     }
     this.addShortcuts()
+
+    const [anticheatInfo, setAnticheatInfo] = useState<AntiCheatInfo | null>(
+      null
+    )
+    window.api
```

Comment:
`window.api...` is meant to be used only in the frontend, you should import `gameAnticheatInfo` directly from `anticheat/utils` since you are already in the backend side